src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -1873,6 +1877,8 @@ trait Contexts { self: Analyzer =>
     private def imp1Explicit = imp1 isExplicitImport name
     private def imp2Explicit = imp2 isExplicitImport name
   }
+
+  private val TupleOfNullAndNoSymbol = (null, NoSymbol)
```

Comment:
```scala
val `(null, NoSymbol)` = (null, NoSymbol)
```
is the classy way to do it.

Reply:
not sure what I just broke, but I'm wondering what it would look like, rendered the classy way:
```
java.lang.AbstractMethodError: Receiver class scala.tools.nsc.interactive.Global$$anon$4 does not define or inherit an implementation of the resolved method abstract scala$tools$nsc$typechecker$Contexts$_setter_$scala$tools$nsc$typechecker$Contexts$$TupleOfNullAndNoSymbol_$eq(Lscala/Tuple2;)V of interface scala.tools.nsc.typechecker.Contexts.
  at scala.tools.nsc.typechecker.Contexts.$init$(Contexts.scala:1881)
```

Reply:
So they only pass bootstrapped. I'll have to look at those tests someday.

Reply:
Which tests are failing for you?

Reply:
They all failed except the `doc` test! with compilation as shown.

Reply:
OK, nm, I did a deep clean and partest --presentation and it passed. Thought I'd done that this morning, reflexively, but obviously not. Recent coronavirus brain fog must be to blame.

Reply:
Maybe this Zinc bug I https://github.com/sbt/zinc/issues/662.

Reply:
Oh that looks familiar, thanks. I was thinking incremental bug, but just couldn't believe it, and had no bandwidth to reproduce. I thought interactive in its own subproject might have triggered something, or failed to trigger.